examples/multi_blade_row_example.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2024 ANSYS, Inc. All rights reserved
+"""
+.. _multi_blade_row_example:
+
+Multi blade row meshing example
+-------------------------------
+This basic example shows how to set up a multi blade row meshing instance and execute it in parallel.
+
+"""
+
+#################################################################################
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Perform the required imports. It is assumed that the ``ansys-turbogrid-core``
+# package has been installed.
+
+import os
+
+from ansys.turbogrid.core.multi_blade_row.multi_blade_row import MultiBladeRow
+
+#################################################################################
+# Create and use an MultiBladeRow instance
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
All the inconsistencies between "multi blade-row" and "multi blade row" have already been resolved as per the previous comments. However, here we have "multi-blade row" compared to "multi blade row" elsewhere. These should be made consistent too.